<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konfiguracja basenu</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ilość filtrów</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Symuluj</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="num_filters">
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Długość symulacji</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="max_timestamp">
      <property name="maximum">
       <number>99999999</number>
      </property>
      <property name="value">
       <number>17520</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Objętość basenu</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="ticket_price">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="pool_volume">
      <property name="maximum">
       <number>9999999</number>
      </property>
      <property name="value">
       <number>2500</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Limit ludzi</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="people_limit">
      <property name="maximum">
       <number>999999</number>
      </property>
      <property name="value">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Cena biletu</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="close_at">
      <property name="maximum">
       <number>23</number>
      </property>
      <property name="value">
       <number>22</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Płaca konserwatora</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="repairman_salary">
      <property name="maximum">
       <number>99999</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Koszt filtra</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="filter_cost">
      <property name="maximum">
       <number>999999</number>
      </property>
      <property name="value">
       <number>15000</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Godzina otwarcia</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="open_at">
      <property name="maximum">
       <number>23</number>
      </property>
      <property name="value">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Godzina zamknięcia</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
